charmcraft/commands/store/__init__.py

```diff
@@ -37,6 +38,30 @@
 LibData = namedtuple(
     'LibData', 'lib_id api patch content content_hash full_name path lib_name charm_name')
 
+LIBRARY_TEMPLATE = """
+\"""TEMPLATE FIXME: Add a proper docstring here.
+
+This is the main documentation of the library, will be exposed by Charmhub after
+the lib is published.
+
+Markdown is supported.
+\"""
+
+# Never change this field, it's the unique identifier to track the library in
+# all systems
+LIBID = "{lib_id}"
+
+# Update this API version when introducing backwards incompatible
+# changes in the library.
```

Comment:
maybe, just maybe mention that this should be incremented by 1 when introducing incompatible changes?

Reply:
FWIW also as written it sounds like you need to increase PATCH every time, even when you just reset API, and I don't think that's the case

Reply:
Nice, I changed both "update" for "increment by 1" (not exactly, better rephrasing).

Reply:
And improved the explanation to show what should happen with PATCH when changing API.